exporter/clickhouseexporter/integration_test.go

```diff
@@ -26,10 +26,11 @@ func TestIntegration(t *testing.T) {
 		name  string
 		image string
 	}{
-		{
-			name:  "test clickhouse 24-alpine",
-			image: "clickhouse/clickhouse-server:24-alpine",
-		},
+		// TODO: Skipping due to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/32530
```

Comment:
I see it fails on `24-apline`, but not sure why this doesn't work when the others do? Is this commented out to reduce test duration?

Reply:
Sorry, I was initially going to comment them out one at a time as they failed, since the integration tests fail as soon as a failure happens. The second run failed on 23, so I just commented them all out. FWIW, they're passing for me locally.

Reply:
These tests that I've commented out are not currently running in main. I've filed issues for them, and now marked them explicitly skipped due to the bug filed. Commenting these out doesn't change any existing functionality, the only difference is now it's documented.

Let me know if this answers your concern, or if I missed something 👍